From 33c02c5aef20cc8bbab09935765f5e5763278233 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 1 Dec 2023 16:03:38 +0100
Subject: [PATCH 0130/1695] plat-stm32mp2: add tamper masking in SYSCFG support

Adds support to access SYSCFG_POTTAMPRSTCR in order to mask some resource
reset upon receiving a potential tamper.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 6846a436f55edcf1efaba6ffecf614f3532a2692 plat-stm32mp2: add tamper masking in SYSCFG support]
Change-Id: I81483d2c2c5d0be7f03440e9692535e95628725e
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/stm32_sysconf.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/stm32_sysconf.h b/core/arch/arm/plat-stm32mp2/stm32_sysconf.h
index 8924ca3e5..954c02f4d 100644
--- a/core/arch/arm/plat-stm32mp2/stm32_sysconf.h
+++ b/core/arch/arm/plat-stm32mp2/stm32_sysconf.h
@@ -26,6 +26,7 @@ enum syscon_banks {
  * SYSCFG register offsets (base relative)
  */
 #define SYSCFG_VDERAMCR		SYSCON_ID(SYSCON_SYSCFG, 0x1800U)
+#define SYSCFG_POTTAMPRSTCR	SYSCON_ID(SYSCON_SYSCFG, 0x1804U)
 
 /*
  * SYSCFG_VDERAMCR register offsets
-- 
2.47.2

